neo/SmartContract/InteropService.cs

```diff
@@ -499,6 +511,15 @@ private static bool Contract_Call(ApplicationEngine engine)
             if (currentManifest != null && !currentManifest.CanCall(contract.Manifest, method.GetString()))
                 return false;
 
+            if (engine.InvocationCounter.TryGetValue(contract.ScriptHash, out var counter))
+            {
+                engine.InvocationCounter[contract.ScriptHash] = counter + 1;
+            }
+            else
+            {
+                engine.InvocationCounter[contract.ScriptHash] = 1;
```

Comment:
Shargon, could this Dictionary be already initialized in 1 and avoid this if?

Reply:
we need to increment the value, so you need to query the previous value

Reply:
Yes, but I mean, for the first time initialize already with 1 and then just increment.

Reply:
This would simplify the if.

Reply:
Initialize to 1 when it is not found